src/includes/performance/enable-tracing/java.spring-boot.mdx

```diff
@@ -20,11 +20,15 @@ sentry.traces-sample-rate=0.3
 Or through providing a bean of type `SentryOptions#TracesSamplerCallback`:
 
 ```java
+import io.sentry.SentryOptions.TracesSamplerCallback;
+import io.sentry.SamplingContext;
+import org.springframework.stereotype.Component;
+
 @Component
-class CustomTracesSamplerCallback {
-  @Override
-  public @NotNull Double sample(@NotNull SamplingContext samplingContext) {
-    // return a number between 0 and 1
-  }
+class CustomTracesSamplerCallback implements TracesSamplerCallback {
+    @Override
+    public Double sample(SamplingContext samplingContext) {
```

Comment:
I'd prefer to keep the `NotNull` or `Nullable` for return and param types

Reply:
It doesnt make copy & paste any easier so I'm against. It may perhaps also suggest for less experienced users that they need extra library..?

Reply:
thats true, good point.